### PR TITLE
Color-polygon functionality and some structure errors fixed

### PR DESCRIPTION
**Pull request by liu-hanli**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -328,3 +331,7 @@ POST Polygon
         [75, 29]
       ]
     }'
+
+With the "format_out" as "geojson" or "polygon", the result will be simple multipoints with the elevation values as 3rd dimension in the coordinates.
+
+With the "format_out" as "colorpolygon", the result will be a feature collection with the elevation values assigned as properties to geometries of polygon or multipolygon.
```
Comment (franchu7) [inline README.rst:337]: Is it necessary to have polygon and colorpolygon? After testing both queries, the 'colorpolygon' query is the most appropriate. The 'polygon' query could be removed.

File: openelevationservice/server/api/querybuilder.py [hunk 2]
```diff
@@ -32,19 +32,95 @@ def _getModel(dataset):
...
+    result_geom = query_final.scalar()
+
+    # Behaviour when all vertices are out of bounds
+    if result_geom == None:
+        raise InvalidUsage(404, 4002,
+                           'The requested geometry is outside the bounds of {}'.format(dataset))
+    
+    return result_geom
+
+
```
Comment (franchu7) [inline openelevationservice/server/api/querybuilder.py:123]: If 'polygon_elevation(geometry, format_out, dataset)' is no longer to be used for anything else, then replace its implementation by that of 'polygon_coloring_elevation'.

File: ops_settings_docker.sample.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
-attribution: "service by https://openrouteservice.org | data by http://srtm.csi.cgiar.org"
-coord_precision: 1e-6
+attribution: "internal service of propamap | data by http://srtm.csi.cgiar.org"
+coord_precision: 0.00083333333333333333
```
Comment (franchu7) [inline ops_settings_docker.sample.yml:3]: This is the exact precision? If not, why is the use of 1e-6 dispensed with?

Comment (carloshellin) [reply ops_settings_docker.sample.yml:3]: I think 1e-6 is the old one, but let the author confirm it.

File: openelevationservice/server/api/querybuilder.py [hunk 4]
```diff
@@ -32,19 +32,95 @@ def _getModel(dataset):
...
+    :raises InvalidUsage: internal HTTP 500 error with more detailed description. 
+        
+    :returns: 3D polygon as GeoJSON or WKT
+    :rtype: string
+    """
+    Model = _getModel(dataset)
+
+    # May be a good idea to make this a parameter from request
+    num_ranges = 23
+    range_div = 43
```
Comment (carloshellin) [inline openelevationservice/server/api/querybuilder.py:67]: Explain at the next meeting why this might be a good idea, and maybe we can consider it.

**Review by carloshellin — APPROVED** [review]
All good

File: README.rst (hunk 1 above)
Comment (franchu7) [reply README.rst:337]: OK

**Pull request merged**